### PR TITLE
Improved HomeFragment

### PR DESCRIPTION
**Pull request by uuilmer**
Was so far able to improve the HomeFragment -> ProfileFragment animation.

### CODE REVIEW

**Comment by uuilmer** [comment]
Streaming Animation:

ezgif-6-fe8e55e4c307

Like Animation:

ezgif-6-0340c2d2b608

**Comment by uuilmer** [comment]
Added relative timing for chats:

116783752_287135596072283_1511073065316604507_n

**Comment by uuilmer** [comment]
App now remembers if you are logged in across app restarts:

ezgif-4-835c59a9123a

File: app/src/main/java/com/example/pix/home/models/Like.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import com.parse.ParseQuery;
+import com.parse.ParseUser;
+
+import java.util.List;
+
+@ParseClassName("Like")
+public class Like extends ParseObject {
+
+    public static final String STREAMER_CODE = "streamer";
+    public static final String LISTENER_CODE = "listener";
```
Comment (xuyueee) [inline app/src/main/java/com/example/pix/home/models/Like.java]: You declared them but didn't use them everywhere in this class.

File: app/src/main/java/com/example/pix/home/adapters/ChatsAdapter.java [hunk 2]
```diff
@@ -93,10 +97,30 @@ public void bind(Chat chat) {
...
+            sf.setLenient(true);
+
+            String relativeDate = "";
+            try {
+                //Get Unix Epoch and get relative from today, then call toString to get readable difference
+                long dateMillis = sf.parse(json_response).getTime();
+                relativeDate = DateUtils.getRelativeTimeSpanString(dateMillis,
+                        System.currentTimeMillis(), DateUtils.SECOND_IN_MILLIS).toString();
+            } catch (java.text.ParseException e) {
+                e.printStackTrace();
```
Comment (xuyueee) [inline app/src/main/java/com/example/pix/home/adapters/ChatsAdapter.java]: Better error handling. It's probably not necessary to print the stack trace, besides for debugging purposes. In this case, probably just roll back to the raw date.

**Review by xuyueee — COMMENTED** [review]
Looks good to me. Please address the comments before merging.

**Pull request merged**